common3.json

```diff
@@ -6879,75 +6879,75 @@
     },
     {
       "id" : 61731,
-      "text" : "拡張ディスク用"
+      "text" : "For Expansion Disk"
     },
     {
       "id" : 61732,
-      "text" : "拡張ディスク用タイトル"
+      "text" : "Title For Expansion Disk"
     },
     {
       "id" : 61733,
-      "text" : "拡張ディスク用トップメニュー"
+      "text" : "Top Menu For Expansion Disk"
     },
     {
       "id" : 61734,
-      "text" : "拡張ディスク用ポケクーポン"
+      "text" : "Poké coupon For Expansion Disk"
     },
     {
       "id" : 61735,
-      "text" : "拡張ディスク用聖なるほこら"
+      "text" : "Relic Stone For Expansion Disk"
     },
     {
       "id" : 61736,
-      "text" : "負け"
+      "text" : "Defeat"
     },
     {
       "id" : 61737,
-      "text" : "あたり無効ボタンON\/OFF"
+      "text" : "Invalid あたり button ON\/OFF"
```

Comment:
This is not in your list, is this translatable?

Reply:
Oh right, this one has two different meanings "Hit" and "Nearby" and I'm not sure which is appropriate here